app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+def copy_file(command: str) -> None:
+    # Split the command into parts
```

Comment:
Remove comments that are self explanatory. If you use split() it is obvious that this is splitting the string